code/modules/robotics/bot/secbot.dm

```diff
@@ -113,6 +116,7 @@
 	idcheck = 1
 	auto_patrol = 1
 	report_arrests = 1
+	move_arrest_delay_mult = 1.2 // beepsky has some experience chasing crimers
```

Comment:
Doesn't this make beepsky slower?
Also just below here, beepsky still has his `attack_per_step = 1`

Reply:
sorry I messed up a partial commit